django/otto/templates/terms_of_use.html

```diff
@@ -43,6 +43,10 @@ <h3 class="h4 mt-4">{% trans "Security, privacy and information management" %}</
     <p>
       {% trans "In the AI assistant, document libraries and chats will be deleted after 30 days of inactivity. This means that if you do not open a chat for 30 days, it will be automatically deleted along with all files uploaded to the chat. For libraries, if no changes have been made and no Q&A queries have been performed for 30 days, the library will be automatically deleted." %}
     </p>
+    <p>
+      {% trans "If information is shared with you through Otto which you should not have access to, use the provided links to email the authors or library administrators. If this is not possible, contact the Otto team at the following email address to report a privacy issue:" %}
```

Comment:
Added paragraph to note the purpose of the email links in the context of privacy issues.